tests/tests/state_rewind.rs

```diff
@@ -251,3 +254,40 @@ async fn rollback_chain_to_same_height_1000() -> anyhow::Result<()> {
     )
     .await
 }
+
+#[tokio::test(flavor = "multi_thread")]
+async fn rollback_to__should_work_with_empty_gas_price_database() -> anyhow::Result<()> {
+    let mut rng = StdRng::seed_from_u64(1234);
+    let driver = FuelCoreDriver::spawn_feeless(&[
+        "--debug",
+        "--poa-instant",
+        "true",
+        "--state-rewind-duration",
+        "7d",
+    ])
+    .await?;
+
+    // Given
+    const TOTAL_BLOCKS: u64 = 500;
+    for _ in 0..TOTAL_BLOCKS {
+        produce_block_with_tx(&mut rng, &driver.client).await;
+    }
+    let temp_dir = driver.kill().await;
+    std::fs::remove_dir_all(temp_dir.path().join("gas_price")).unwrap();
+
+    // When
+    let args = [
+        "_IGNORED_",
```

Comment:
I've seen this being used in other tests too, what does this mean?

Reply:
That's the first (shell) command argument, which is usually the path to the binary. Since the executable name isn't used for anything and pointing to the real fuel-core binary would make no sense, just having a dummy string here is fine.